src/devtools/client/inspector/rules/actions/rules.ts

```diff
@@ -1,66 +0,0 @@
-import { assert } from "protocol/utils";
-import type { UIStore } from "ui/actions";
-import { isInspectorSelected } from "ui/reducers/app";
-import { AppStartListening } from "ui/setup/listenerMiddleware";
```

Comment:
I think you probably meant to delete `/reducers/rules.ts` and `/selectors/rules.ts` too, but forgot to?